src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6020,14 +6070,14 @@ namespace SimAirServingZones {
 					PreDefTableEntry( pdchS62svrHtEv, FinalSysSizing( AirLoopNum ).AirPriLoopName, EvzMinBySysHeat( AirLoopNum ), 3 ); //Ev
 					PreDefTableEntry( pdchS62svrHtVot, FinalSysSizing( AirLoopNum ).AirPriLoopName, VotHtgBySys( AirLoopNum ), 4 ); //Vot
 					if ( FinalSysSizing( AirLoopNum ).DesHeatVolFlow != 0.0 ) { // Move here from other routine
-						PreDefTableEntry( pdchS62svrHtPercOA, FinalSysSizing( AirLoopNum ).AirPriLoopName, VotHtgBySys( AirLoopNum ) / FinalSysSizing( AirLoopNum ).DesHeatVolFlow ); //%OA
+						PreDefTableEntry( pdchS62svrHtPercOA, FinalSysSizing( AirLoopNum ).AirPriLoopName, 100.0 * VotHtgBySys( AirLoopNum ) / FinalSysSizing( AirLoopNum ).DesHeatVolFlow ); //%OA
```

Comment:
I realize it says percent in the title, but the report's been showing a fraction value for quite some time now and nobody has complained. So we need to find out what the preference is.  

@kbenne @macumber Does OpenStudio grab values from the Standard 62.1 report?  Look at the far right column, title says "Percent Outdoor Air - %OA" but the values are reported as a fraction (0 - 1).  Change title to "Fraction" or change values to percent (0 - 100)?
![image](https://cloud.githubusercontent.com/assets/4991075/11156662/d5425776-8a12-11e5-84b2-49c1739c2094.png)